src/dirs.proj

```diff
@@ -12,6 +12,22 @@
     <Warning Text="Looks like there may be an update to the build tools. Please run build.cmd from the root of the repo to refresh the build tools." /> 
   </Target>
 
+  <ItemGroup>
+    <!-- Include test runtime when restoring packages in EnsureDependencies -->
+    <PackageConfigs Include="$(BaseIntermediateOutputPath)test-runtime\packages.config" />
+  </ItemGroup>
+  
+  <Target Name="PrepareTestRuntimeRestore" BeforeTargets="EnsureDependencies">
+    <Copy
+      SourceFiles="$(ToolsDir)test-runtime-packages.config"
+      DestinationFiles="$(BaseIntermediateOutputPath)test-runtime\packages.config"
+      SkipUnchangedFiles="$(SkipCopyUnchangedFiles)"
+      OverwriteReadOnlyFiles="$(OverwriteReadOnlyFiles)"
+      Retries="$(CopyRetryCount)"
+      RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
+      UseHardlinksIfPossible="$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)" />
+  </Target>
```

Comment:
I could do that, the reason I did this was to preserve the name "test-runtime" with the file to describe its purpose and keep an association between the file deployed and the file in source control.  What do you think about that vs. the simplicity of just renaming it in the nuspec?